credentials/settings/base.py

```diff
@@ -583,4 +583,16 @@
             "enabled": False,
         },
     },
+    "org.openedx.learning.course.grade.now.passed.v1": {
```

Comment:
These events should go to edx-platform settings.
Instead here badges events are expected.
@kyrylo-kh